tests/run-make/raw-dylib-stdcall-ordinal/rmake.rs

```diff
@@ -0,0 +1,36 @@
+// `raw-dylib` is a Windows-specific attribute which emits idata sections for the items in the
+// attached extern block,
+// so they may be linked against without linking against an import library.
+// To learn more, read https://github.com/rust-lang/rfcs/blob/master/text/2627-raw-dylib-kind.md
+// Almost identical to `raw-dylib-link-ordinal`, but with the addition of calling conventions,
+// such as stdcall.
+// See https://github.com/rust-lang/rust/pull/90782
+
+//@ only-x86
+//@ only-windows
+
+use run_make_support::{cc, diff, is_msvc, run, rustc};
+
+// NOTE: build_native_dynamic lib is not used, as the special `def` files
+// must be passed to the CC compiler.
+
+fn main() {
+    rustc().crate_type("lib").crate_name("raw_dylib_test").input("lib.rs").run();
+    rustc().crate_type("bin").input("driver.rs").run();
+    if is_msvc() {
+        cc().arg("-c").out_exe("exporter").input("exporter.c").run();
+        cc().input("exporter.obj")
+            .arg("exporter-msvc.def")
+            .args(&["-link", "-dll", "-noimplib", "-out:exporter.dll"])
+            .run();
+    } else {
+        cc().arg("-v").arg("-c").out_exe("exporter.obj").input("exporter.c").run();
+        cc().input("exporter.obj")
+            .arg("exporter-gnu.def")
+            .arg("-shared")
+            .output("exporter.dll")
+            .run();
+    };
+    let out = run("driver").stdout_utf8();
+    diff().expected_file("output.txt").actual_text("actual", out).normalize(r#"\r"#, "").run();
```

Comment:
Question: this `output.txt` is a bit suspicious to me, are we sure this isn't supposed to be `expected_output.txt`?

The other test `tests/run-make/raw-dylib-link-ordinal` *does* in fact have a `output.txt`, but in *this* test directory there's only `expected_output.txt`.

Or rather, how did this not fail the try job?

Reply:
`x86_64-mingw` and `x86_64-msvc`: `raw-dylib-stdcall-ordinal ... ignored, only executed when the architecture is x86`

????????????????

Reply:
Problem: actually, sorry I didn't quite register this: if the original test was `//@ only-x86`, then this is not more sensible than `//@ only-x86_64`, because according to compiletest `only-*` directive matching precedence, the matcher that takes precedence for our cases is target architecture, so in this sense `x86` is the 32-bit x86 architecture while `x86_64` is the 64-bit x86_64 architecture.